### DISCUSSION TITLE
zeekloss.sh and oldpcap.sh log errors

### DISCUSSION THREAD

**Discussion by trocade** [Security-Onion-Solutions/securityonion#6976] [Unsupported Versions] [answered]
I was looking at logs on my sensors and noticed that telegraf are logging these errors quite often:

`2022-01-25T16:01:30Z E! [inputs.exec] Error in plugin: exec: exit status 1 for command '/scripts/oldpcap.sh': /scripts/oldpcap.sh: line 25: 1643126490 - : syntax error: operand expected (error token is "- ")`
`2022-01-25T16:01:30Z E! [inputs.exec] Error in plugin: exec: exit status 1 for command '/scripts/zeekloss.sh': /scripts/zeekloss.sh: line 42: 5.69591e+09: syntax error: invalid arithmetic operator (error token is ".69591e+09")`

is it something to care about?
Currently at 2.3.70 but going to upgrade to latest release this week.

**Comment by dougburks** [comment, marked as answer]
I'd go ahead and upgrade to the latest, reboot, and see if that helps.

**Comment by trocade** [comment]
After upgrade to 2.3.91 the error is no longer there (up and running with 2.3.91 for approx. 12 hours)
